nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md

```diff
@@ -0,0 +1,22 @@
+# Braket-SDK [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This node type installs Braket SDK.
+
+> **⚠ Executions may not be reproducible!** This Node Type installs Braket SDK in its latest version.
+> Thus, applications that depend on this Node Type may not be compatible with the installed version.
```

Comment:
Maybe it's better then to install a specific version?

Did you add an example app that demonstrates it's usage to the example apps repo?

Reply:
Installing a specific version could also cause troubles with reproducibility, as it's likely outdated when the next guy wants to use this node type. Is it possible to draw the version from a property and install latest just in the case that no specific version has been set via that property?

I guess the example app should be added to [tosca-definitions-qc-applications](https://github.com/UST-QuAntiL/tosca-definitions-qc-applications)?

Reply:
Okay, then just add this explanation to the readme.

Yes, example apps should be in the qc-apps repo.